_extras/design.md

```diff
@@ -118,10 +118,84 @@ How do I...
 ### Variables and Assignment (9:15)
 
 *   Teaching: 10 min
+
+Basic assignment of a value.  What kind of thing is it?  Can assign results
+of computation and use a variable on the right to assign to the left.
+
+```
+a = 3
+print(a)
+type(a)
```

Comment:
This won't show up when people run the code unless it's the last line - please use `print(type(a))`